commons/src/main/java/nl/nn/adapterframework/util/DateFormatUtils.java

```diff
@@ -44,28 +45,40 @@ public class DateFormatUtils {
 
 
 	public static String now() {
-		return format(new Date());
+		return format(Instant.now());
 	}
 
 	@Deprecated
 	public static String now(String format) {
-		return format(new Date(), new SimpleDateFormat(format));
+		return format(Instant.now(), new SimpleDateFormat(format));
 	}
 
-	public static String format(long date) {
-		return format(new Date(date));
+	public static String format(Date date) {
+		return format(date.getTime());
 	}
 
-	public static String format(Date date) {
+	public static String format(long date) {
 		return format(date, FULL_GENERIC_FORMATTER);
 	}
 
-	public static String format(Date date, DateFormat formatter) {
+	public static String format(Instant instant) {
+		return format(instant.toEpochMilli(), FULL_GENERIC_FORMATTER);
+	}
+
+	public static String format(long date, DateFormat formatter) {
 		return formatter.format(date);
 	}
 
+	public static String format(long date, String format) {
```

Comment:
After making another comment later I want to add that use of this method should be absolutely minimised since creating any DateFormat instance of any API is expensive. They should be reused, not thrown away after single use.

HOWEVER the old API which you use here is not threadsafe so really, do please replace this with the new java.time API at the same time as you remove use of this method where possible.